.pre-commit-config.yaml

```diff
@@ -66,4 +66,4 @@ repos:
   #       language: python
   #       entry: python scripts/component_readme/generate_readme.py
   #       files: ^components/.*/fondant_component.yaml
-  #       additional_dependencies: ["fondant"]
\ No newline at end of file
+  #       additional_dependencies: ["fondant[aws,azure,gcp]@git+https://github.com/ml6team/fondant@main"]
```

Comment:
Can you exclude this file from the PR?

Reply:
Sure. Keep in mind. Without this I wasnt able to creat READMEs that passed the ci/cd pipeline. I needed the latest fondant version.